### PR TITLE
 Match UI elements to game globals.

### PR DESCRIPTION
**Pull request by phuein**
Replaces the old UI reload warning elements' text with the game defaults. Also removes the red coloring in the warning text, back to the default tooltip text color.

Do note that the game implies a UI reload won't **_necessarily_** happen, which is bizarre. Makes me wonder if the addon's old texts made more sense. For example, the warning from the game is:

> This setting must be applied and may result in a load screen.

The "Reload UI" button is then titled "Apply" instead.

### CODE REVIEW

File: LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua [hunk 1]
```diff
@@ -349,9 +349,9 @@ local localization = {
         TRANSLATION = "Translation",
         DONATION = "Donate",
         PANEL_INFO_FONT = "$(CHAT_FONT)|14|soft-shadow-thin",
-        RELOAD_UI_WARNING = "Changes to this setting require an UI reload in order to take effect.",
-        RELOAD_DIALOG_TITLE = "UI Reload required",
-        RELOAD_DIALOG_TEXT = "Some changes require an UI reload in order to take effect. Do you want to reload now or discard the changes?",
+        RELOAD_UI_WARNING = GetString(SI_OPTIONS_APPLY_WARNING),
+        RELOAD_DIALOG_TITLE = GetString(SI_ADDON_MANAGER_RELOAD),
+        RELOAD_DIALOG_TEXT = "Some changes require a UI reload in order to take effect. Do you want to reload now or discard the changes?",
```
Comment (sirinsidiator) [inline LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua]: Are you sure about this? I am not a native speaker, but I am pretty certain that it is "an UI reload".

File: LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua [hunk 2]
```diff
@@ -349,9 +349,9 @@ local localization = {
         TRANSLATION = "Translation",
         DONATION = "Donate",
         PANEL_INFO_FONT = "$(CHAT_FONT)|14|soft-shadow-thin",
-        RELOAD_UI_WARNING = "Changes to this setting require an UI reload in order to take effect.",
-        RELOAD_DIALOG_TITLE = "UI Reload required",
-        RELOAD_DIALOG_TEXT = "Some changes require an UI reload in order to take effect. Do you want to reload now or discard the changes?",
+        RELOAD_UI_WARNING = GetString(SI_OPTIONS_APPLY_WARNING),
```
Comment (sirinsidiator) [inline LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua]: I just took a look at it ingame and you may be right. It feels strange to read "may result in a loading screen" when it does require one for sure. Please revert these changes and just leave the original text. Otherwise we'd have to update all the translations too. We can think about a better text for LAM3 in a separate ticket.

File: LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua (hunk 1 above)
Comment (phuein) [reply LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua]: I am sure. The rule is "how it sounds" not "which letter." For any English speaker, reading that aloud will sound very wrong.

https://www.englishforums.com/English/AUtilityVsAnUtility/hrmlw/post.htm

File: LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua (hunk 2 above)
Comment (phuein) [reply LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua]: Fair enough. I feel the same. I wonder if there's an easy way to do this with GitHub, without redoing the whole pull req.

File: LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua (hunk 1 above)
Comment (sirinsidiator) [reply LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua]: Guess you are right. The more you know. :D

File: LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua (hunk 2 above)
Comment (sirinsidiator) [reply LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua]: Just create another commit with the original text. I will squash it on merge anyways, so you won't see the individual commits, but only the end result.

**Comment by phuein** [comment]
Looks like my latest commit with text updates is applied here automagickally! So this can/should be accepted.
